pkg/utils/config/config_define.go

```diff
@@ -19,15 +19,16 @@ import "time"
 type Default struct{}
 
 type OsdsApiServer struct {
-	ApiEndpoint         string        `conf:"api_endpoint,localhost:50040"`
-	AuthStrategy        string        `conf:"auth_strategy,noauth"`
-	PolicyPath          string        `conf:"policy_path,/etc/opensds/policy.json"`
-	Daemon              bool          `conf:"daemon,false"`
-	LogFlushFrequency   time.Duration `conf:"log_flush_frequency,5s"` // Default value is 5s
-	HTTPSEnabled        bool          `conf:"https_enabled,false"`
-	BeegoHTTPSCertFile  string        `conf:"beego_https_cert_file,/opt/opensds-security/opensds/opensds-cert.pem"`
-	BeegoHTTPSKeyFile   string        `conf:"beego_https_key_file,/opt/opensds-security/opensds/opensds-key.pem"`
-	PasswordDecryptTool string        `conf:"password_decrypt_tool,aes"`
+	ApiEndpoint        string        `conf:"api_endpoint,localhost:50040"`
+	Graceful           bool          `conf:"graceful,true"`
```

Comment:
Please remove `Graceful` and `SocketOrder` these two fileds.

Reply:
Ask @xxwjj

Reply:
Has been deleted.